app/code/Magento/Developer/Console/Command/TemplateHintsStatusCommand.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Developer\Console\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+
+/**
+ * Command to show frontend template hints status
+ */
+class TemplateHintsStatusCommand extends Command
+{
+
+    /**
+     * command name
+     */
+    const COMMAND_NAME = 'dev:template-hints:status';
+
+    /**
+     * Success message
+     */
```

Comment:
Comments are not necessary to const

Reply:
I've copied from magento2/app/code/Magento/Developer/Console/Command/TemplateHintsEnableCommand.php (Core)

Reply:
It means that core also needs to be fixed :-) We improved Quality standards in the meantime and as a result - something that passed QA before, would not pass now.

Reply:
You're right!
But now, I've done all the changes needed.
I hope this is the right time.